DTS-STN/canadian-dental-care-plan
# AB#3303 -- Updated english/french translations for the apply flow

### Description
Updated english/french translations for the apply flow 

### Related Azure Boards Work Items
[AB#3303](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/3303)

### Screenshots (if applicable)
n/a

### Checklist

### Test Instructions
View apply flow pages to confirm they conform with the May release final figma design

### Additional Notes
n/a
